core/domain/subtopic_page_services_test.py

```diff
@@ -33,17 +33,6 @@
     models.NAMES.base_model, models.NAMES.topic])
 
 
-# TODO(lilithxxx): Remove this mock class and the migrate_page_contents tests
-# once the actual functions for page content migrations are implemented.
```

Comment:
Why removing this?

Reply:
same as this [comment](https://github.com/oppia/oppia/pull/9350#discussion_r430671921)